Avoid exec/SIGCHLD race in daemon fork.
Let's see if it fixes race seen in https://bugs.gentoo.org/727344
--- a/Makefile
+++ b/Makefile
@@ -280,7 +280,7 @@ test_elixir: test_stdlib test_ex_unit test_logger test_mix test_eex test_iex
 
 test_stdlib: compile
 	@ echo "==> elixir (ex_unit)"
-	$(Q) exec epmd & exit
+	$(Q) epmd -daemon
 	$(Q) if [ "$(OS)" = "Windows_NT" ]; then \
 		cd lib/elixir && cmd //C call ../../bin/elixir.bat -r "test/elixir/test_helper.exs" -pr "test/elixir/**/$(TEST_FILES)"; \
 	else \
